api/server/openapi_info.go

```diff
@@ -0,0 +1,23 @@
+package server
+
+// General OpenAPI/Swagger metadata for the SSV HTTP API.
+//
+// @title           SSV Node API
+// @version         0.1.0
+// @description     HTTP endpoints exposed by the SSV node, including exporter traces and validator metadata.
+// @contact.name    SSV Labs
+// @contact.url     https://ssv.network
+// @contact.email   support@ssv.network
```

Comment:
ssv labs is at ssvlabs.io , ssv.network is the address of ssv.network DAO or proejct. 

short: change to ssvlabs.